src/udiskslinuxfilesystem.c

```diff
@@ -139,50 +139,19 @@ udisks_linux_filesystem_new (void)
 
 /* ---------------------------------------------------------------------------------------------------- */
 
-/**
- * udisks_linux_filesystem_update:
- * @filesystem: A #UDisksLinuxFilesystem.
- * @object: The enclosing #UDisksLinuxBlockObject instance.
- *
- * Updates the interface.
- */
-void
-udisks_linux_filesystem_update (UDisksLinuxFilesystem  *filesystem,
-                                UDisksLinuxBlockObject *object)
+static guint64
+get_filesystem_size (UDisksLinuxBlockObject *object)
 {
-  UDisksMountMonitor *mount_monitor;
+  guint64 size = 0;
+  GError *error = NULL;
```

Comment:
I think you may remove all uses of `error` in this function as we don't use it and the `bd_fs_xxx_get_info()` calls may consume _NULL_ just fine.

Reply:
Yes, sure, I just moved it so I didn't think about changing it.